assets/app/views/services.html

```diff
@@ -16,7 +16,12 @@
         <div>Container port: {{service.spec.containerPort}}</div>
         <div>Protocol: {{service.spec.protocol}}</div>
         <div>Selectors:<span ng-repeat="(selectorLabel, selectorValue) in service.spec.selector"> {{selectorLabel}}={{selectorValue}}</span></div>
-      </div>      
+        <div>Routes: <span ng-repeat="route in routes[service.metadata.name]"> {{route.host}}</span></div>
```

Comment:
`ng-repeat="(routeId, route) in routes[serviceId]"` (and change the `ng-repeat` above to `(serviceId, service) in services`)